src/docs/reference/modules/Users/README.md

```diff
@@ -0,0 +1,19 @@
+# Users (OrchardCore.Users)
+
+The Users module enables authentication UI and user management.
+
+## Custom Paths
+
+If you want to specify custom paths to login page, logoff, etc...
```

Comment:
If you want to specify custom paths to access the authentication related urls, you can change ithem by using this option in the appsettings.json: